Packs/MongoDB/Integrations/MongoDB/README.md

```diff
@@ -87,7 +88,7 @@ Searches items by query
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
 | collection | Name of the collection do query from. | Required | 
-| query | A JSON query to search in collection. | Required | 
+| query | A JSON query to search for in the collection, in the format of: `{"key": "value"}`. e.g {"_id": "mongodbid"}. Supports search by regex using the following query=`"{ "field": { "$regex": "search_option" } }"`. For example: query=`"{ "year": { "$regex": "2.*" } }"` - will query all entries such that their "year" field contains the number 2, query=`"{ "color": { "$regex": "Re.*", "$options": "i" } }"`: case insensitive search - will query all entries at the collection, where their "color" field contains the string "Re", c.| Required |
```

Comment:
is the `, c.` on purpose also again make sure this is in line with what is written in the yml descriptions.